tests/FilledStringTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Pinnacle\CommonValueObjects\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Pinnacle\CommonValueObjects\FilledString;
+use UnexpectedValueException;
+
+class FilledStringTest extends TestCase
+{
+    /**
+     * @test
+     */
+    public function hasEmptyString_ThrowsException()
+    {
+        $this->expectException(UnexpectedValueException::class);
+        new FilledString('');
+    }
+
+    /**
+     * @test
+     */
+    public function hasStringWithSpaces_ThrowsException()
+    {
+        $this->expectException(UnexpectedValueException::class);
+        new FilledString('          ');
+    }
+
+    /**
+     * @test
+     */
+    public function hasStringWithContent_DoesNotThrowException()
+    {
+        $string = new FilledString('abc');
+        $this->assertSame('abc', $string->getValue());
+        $this->assertSame('abc', (string)$string);
+    }
+
+    /**
+     * @test
+     */
+    public function hasStringWithContentAndSpaces_DoesNotThrowException()
+    {
+        $string = new FilledString('abc');
```

Comment:
Hmm, wouldn't this test fail? Don't we need to:
```suggestion
        $string = new FilledString('  abc  ');
```